lib/rails-jquery-autocomplete/orm/active_record.rb

```diff
@@ -50,13 +50,15 @@ def get_autocomplete_select_clause(model, method, options)
       def get_autocomplete_where_clause(model, term, method, options)
         table_name = model.table_name
         is_full_search = options[:full]
-        like_clause = (postgres?(model) ? 'ILIKE' : 'LIKE')
+        is_case_sensitive_search = options[:case_sensitive]
+        like_clause = (postgres?(model) && !is_case_sensitive_search ? 'ILIKE' : 'LIKE')
+        column_transform = is_case_sensitive_search ? '' : 'LOWER'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.